src/domain/playGame.js

```diff
@@ -0,0 +1,23 @@
+import playRound from "./playRound.js";
+import { printResultHeader, printRoundScore } from "../view/output.js";
+
+// ['A', 'B', 'C']
+// 3
+
+const playGame = (carNames, rounds) => {
+  let cars = [];
+  for (let name of carNames) {
+    cars.push({ name, count: 0 });
+  }
+
+  printResultHeader();
+  for (let i = 0; i < rounds; i++) {
+    const updatedCars = playRound(cars);
+    cars = updatedCars;
+    // 각 라운드 결과 출력
+    printRoundScore(cars);
+  }
```

Comment:
다음과 같이 축약할 수 있을 것 같습니다. 

```suggestion
  printResultHeader();
  
  let cars = carNames.map((name) => ({ name, count: 0 }));

  Array.from({ length: rounds }).forEach(() => {
    cars = playRound(cars);
    printRoundScore(cars);
  });
```